### PR TITLE
Unify and fix all python libraries version among all python projets

### PR DESCRIPTION
**Pull request by GuillaumeLescure**
Don't hesite to say if you like the idea or not.

The main idea is to create one unified python env for navitia build with :
`mkvirtualenv -r ./eitri/requirements.txt -r ./cities/requirements.txt -r ./tyr/requirements_dev.txt -r ./monitor/requirements.txt -r ./jormungandr/requirements_dev.txt -r ./sql/requirements.txt -p /usr/bin/python2 navitia_dev`

Then just :
`workon navitia_dev`

### CODE REVIEW

File: source/jormungandr/requirements.txt [hunk 1]
```diff
@@ -14,8 +14,8 @@ Werkzeug==0.14.1
 blinker==1.3
 dict2xml==1.1
 itsdangerous==0.24
-protobuf
-psycopg2
+protobuf==3.9.0
```
Comment (kinnou02) [inline source/jormungandr/requirements.txt]: no, we want to keep the version available on the system, that why we do not fix the version.

**Review by kinnou02 — CHANGES_REQUESTED** [review]

File: source/jormungandr/requirements.txt (hunk 1 above)
Comment (GuillaumeLescure) [reply source/jormungandr/requirements.txt]: Ok, done

**Comment by GuillaumeLescure** [comment]
Test this please.

**Comment by GuillaumeLescure** [comment]
Jenkins, test this please

**Comment by mbounabi** [comment]
I don't undertand what keeps you from doing the whole navitia env before this PR? Why upgrading some packages only?
**UPDATE**: Answer -> To fix inconsistencies between projects

**Review by mbounabi — APPROVED** [review]

**Review by AurelienLP — APPROVED** [review]

**Comment by GuillaumeLescure** [comment]
Jenkins, test this please

**Comment by GuillaumeLescure** [comment]
Jenkins, test this please.

**Pull request merged**